notebooks/custom_kinfraglib/1_3_custom_filters_synthesizability.ipynb

```diff
@@ -0,0 +1,2349 @@
+{
```

Comment:
<p>What are the first two variables needed for?</p><p>Actually also, what exactly is contained in fragment_library_enamine?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sonjaleo/KinFragLib/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>